src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+export type Shape = 'circle' | 'triangle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
```

Comment:
Just a suggestion -> You can use `enum` type here.

Reply:
Works like a charm. thanks